test/bigtest/interactors/line-edit-page.js

```diff
@@ -88,6 +90,12 @@ import { ACCORDION_ID } from '../../../src/components/POLine/const';
   funds = collection('[data-test-repeatable-field] [class*=repeatableFieldItem---]');
 }
 
+@interactor class ItemDetailsAccordion {
+  static defaultScope = '#itemDetails';
+  inputTitle = fillable('[name="title"]');
```

Comment:
`'[name="title"]'` can be a constant

Reply:
Yes, I agree with you, but which place for constant do you see? I see two options inside and outside the interactor and both of them I don't like. Inside interactor we'll need to access the constant like a this. ..., if we move it outside the one it would like we need to do the same for all selectors

Reply:
as you said before you like constants, here you used `'[name="title"]'` string twice

I don't see any problems to have it outside, I can't forbid you to update all other places if you want :) but it's not required

Reply:
fixed